src/App.tsx

```diff
@@ -1,14 +1,50 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getAllTodo } from './services/allTodo';
+import { getCompletedTodo } from './services/completedTodo';
+import { getActiveTodo } from './services/activeTodo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [optionsValue, setOptionsValue] = useState('all');
+  const [inputValue, setInputValue] = useState('');
+  const [hasTodoClick, setHasTodoClick] = useState(false);
+  const [userId, setUserId] = useState(0);
+  const [todoId, setTodoId] = useState(0);
+  const [loading, setLoading] = useState(false);
+
+  const fillteredTodos = useMemo(() => {
+    return [...todos].filter(todo =>
+      todo.title.toLowerCase().includes(inputValue.toLowerCase()),
+    );
+  }, [todos, inputValue]);
```

Comment:
The variable 'fillteredTodos' is misspelled. It should be 'filteredTodos' to correctly reflect its purpose of storing filtered todo items.